modules/auth/oauth2/oauth2.go

```diff
@@ -5,16 +5,16 @@
 package oauth2
 
 import (
-	"code.gitea.io/gitea/modules/setting"
 	"code.gitea.io/gitea/modules/log"
+	"code.gitea.io/gitea/modules/setting"
 	"github.com/gorilla/sessions"
 	"github.com/markbates/goth"
 	"github.com/markbates/goth/gothic"
+	"github.com/markbates/goth/providers/github"
+	"github.com/satori/go.uuid"
 	"net/http"
```

Comment:
That's not my code, this change is anyway only there because it have not been formatted